### PR TITLE
Translate message "Nothing to show"

### PR DESCRIPTION
**Pull request by abaliunov-sc**
#305

### CODE REVIEW

File: packages/client-react/src/translations.js [hunk 1]
```diff
@@ -39,6 +43,7 @@ const translations = {
 };
 
 export default function getMessage(locale, key, params) {
+  console.log(locale, key, params);
```
Comment (achernyakevich-sc) [inline packages/client-react/src/translations.js:46]: I'm not sure that we need to log it for every function call. Maybe remove it?

File: packages/client-react/src/translations.js [hunk 2]
```diff
@@ -2,10 +2,14 @@ const translations = {
   en: {
     'common.Toolbar.moveForward': 'Move forward',
     'common.Toolbar.moveBack': 'Move back',
+    'nothingToShow': 'Nothing to show',
```
Comment (ddivin-sc) [inline packages/client-react/src/translations.js:5]: Please, rename the key to 'common.NoFilesFoundStub.title' according to convention in the issue https://opuscapita.atlassian.net/browse/EPROC-20549

File: packages/client-react/src/translations.js [hunk 3]
```diff
@@ -2,10 +2,14 @@ const translations = {
...
+    'useContextMenu': 'Use toolbar or context menu to perform available actions'
```
Comment (ddivin-sc) [inline packages/client-react/src/translations.js:6]: Please, rename the key to 'common.NoFilesFoundStub.message' according to convention in the issue https://opuscapita.atlassian.net/browse/EPROC-20549

**Review by ddivin-sc — COMMENTED** [review]
Please, fix my claims

File: packages/client-react/src/translations.js (hunk 3 above)
Comment (abaliunov-sc) [reply packages/client-react/src/translations.js]: Done

**Review by ddivin-sc — APPROVED** [review]

**Review by estambakio-sc — APPROVED** [review]
LGTM

Consider adding translations for other languages too https://github.com/OpusCapita/filemanager/blob/4c6ea274d199a3e15d163ce1bb86fb5bf3146631/packages/client-react/src/translations.js or rename this PR to `Translate message "Nothing to show" into French`

**Comment by estambakio-sc** [comment]
Translated into Fr, De, Ru. Messages in other languages will be added by @amotuz-sc later.

**Pull request merged**